app/controlplane/internal/service/cascredential.go

```diff
@@ -99,7 +100,11 @@ func (s *CASCredentialsService) Get(ctx context.Context, req *pb.CASCredentialsS
 			mapping, err = s.casMappingUC.FindCASMappingForDownloadByUser(ctx, req.Digest, currentUser.ID)
 			// otherwise, we'll try to find a mapping for the current API token associated orgs
 		} else if currentAPIToken != nil {
-			mapping, err = s.casMappingUC.FindCASMappingForDownloadByOrg(ctx, req.Digest, []string{currentOrg.ID})
+			orgID, err := uuid.Parse(currentOrg.ID)
+			if err != nil {
+				return nil, handleUseCaseErr(err, s.log)
+			}
+			mapping, err = s.casMappingUC.FindCASMappingForDownloadByOrg(ctx, req.Digest, []uuid.UUID{orgID}, nil)
```

Comment:
add a TODO to indicate that we'll pass the projectID once it's supported in API tokens